contracts/CommunityResolver.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {SchemaResolver} from "@ethereum-attestation-service/eas-contracts/contracts/resolver/SchemaResolver.sol";
+import {IEAS} from "@ethereum-attestation-service/eas-contracts/contracts/IEAS.sol";
+import {Attestation} from "@ethereum-attestation-service/eas-contracts/contracts/Common.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract CommunityResolver is SchemaResolver, Initializable, OwnableUpgradeable {
+    mapping(address => uint8) private communityAdmins;
+    // can add community admins
+    mapping(address => uint8) private admins;
```

Comment:
this way, the only one allowed to create a community is the owner (we don't have access to uid before attestation finishes), meaning that first you'll need to attest the community to then add community admins. We'll also need another contract to handle community details because we'll need to check if that attester is a community admin.